src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0"?>
+<xsl:stylesheet
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  exclude-result-prefixes="xsl"
+  version="1.1">
+
+  <xsl:output method="xml" indent="yes" encoding="UTF-8" />
+
+  <xsl:template match="para">
+    <p>
+      <xsl:apply-templates />
+    </p>
+  </xsl:template>
+
+  <xsl:template match="b">
+    <strong>
+      <xsl:apply-templates />
+    </strong>
+  </xsl:template>
```

Comment:
in CAPS, \<b\> would be transformed to \<strong\> eventually. not sure whether there is any css setting in the two tags